server/csp.ts

```diff
@@ -0,0 +1,64 @@
+// CSP eller Content-Security-Policy er en HTTP-Header som lar oss spesifisere hvor appen kan kjøre REST-kall mot og hvor den kan hente diverse innhold fra (fonter, bilder, javascript, stylesheets mm).
```

Comment:
Hvordan er det med fjernhjelp og csp? Vi har noe triks i ludo på serveren for å håndtere det iallefall - er det noe vi må tenke på her også?

Reply:
Det var e.g "'unsafe-eval'" for vergic, og 'worker-src' for blobs ?

Reply:
Men burde vi fjerne X-Frame-Options lik DENY pga fjernhjelp ?

Reply:
Lar den stå og ser hvordan det går med testingen